### ISSUE TITLE
Does not support multiple types in one clip

### ISSUE THREAD

**Issue by rocketraman** [jongrim/react-clipboard-polyfill#4]
Your function `copyData` looks like this:

```
  const copyData = (dataType, data) => {
    const dt = new clipboard.DT();
    dt.setData(dataType, data);
    return clipboard.write(dt);
  }
```

This leaves no possibility for putting both `text/plain` and `text/html` on the clipboard in one DataTransferItem.

You can see an example of this in the upstream polyfill: https://github.com/lgarron/clipboard-polyfill#other-data-types-eg-html

**Comment by jongrim** [comment]
Thanks for posting! I will take a look at possible ways to enable this. I'm thinking I could add a new prop, maybe called `multidata`, that would be an array of objects, each having a `dataType` and `data` property. Then, in `copyData`, I can check for this array, and loop over it setting the multiple types. Does that sound like a reasonable use?

I might also go ahead and enable a way to get the underlying clipboard, so if there is anything else that can be done with it that I haven't covered, users could build it themselves and not be restricted by this component.

**Comment by jongrim** [comment]
Actually, after checking this, I don't think the example is showing that it's possible to set multiple items at once. I think you're limited to setting one type at a time. Check out this [sandbox](https://codesandbox.io/s/r57l41lyrn)

Feel free to reopen if you can provide an example of how to set multiple items at once.

**Comment by rocketraman** [comment]
@jongrim Actually that sandbox does set both text/plain and text/html. After the "Click me" in that sandbox, I can do this (on Linux):

```
$ xclip -selection clipboard -o -t text/html
<meta http-equiv="content-type" content="text/html; charset=utf-8"><b>test</b>
```

```
$ xclip -selection clipboard -o -t text/plain
test
```

which shows you that both mime types have been sent to my clipboard.

If you're on Windows I don't know how you can check the available mime types of clipboard entries, but I'm sure there is a way.

**Comment by rocketraman** [comment]
Actually, it's pretty easy: if you copy from that sandbox and paste into an application that supports HTML (like Google Docs or Word), you'll see the bold test, and if you paste it into an application that does not (like a text editor), you'll see just test.

**Comment by jongrim** [comment]
Interesting, I guess I was expecting it would paste **both** the plain text version and the HTML version. Thanks for explanation, I should have this updated soon 👍

**Comment by jongrim** [comment]
This should be available now after merging #5, and I've published a new version `1.1.0`. If you have any troubles with it let me know.
